zerver/tests/test_messages.py

```diff
@@ -4135,7 +4135,7 @@ def send_fake_message(message_content: str, stream: Stream) -> Message:
         self.assertNotEqual(idle_user_msg_list[-1], sent_message)
         with queries_captured() as queries:
             add_missing_messages(long_term_idle_user)
-        self.assert_length(queries, 6)
+        self.assert_length(queries, 7)
```

Comment:
We should figure out what the new query is -- it likely reflects a bug somewhere.